--- exercises/047_methods.zig	2026-06-09 11:09:29.596972673 +0930
+++ answers/047_methods.zig	2026-06-09 11:14:20.617289233 +0930
@@ -88,7 +88,7 @@
 
             if (alien.health > 0) {
                 // *** Zap living aliens with the heat ray here! ***
-                ???.zap(???);
+                heat_ray.zap(alien);
 
                 // If the alien's health is 0 or below, it's not alive.
                 if (alien.health <= 0) aliens_alive -= 1;
